src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,35 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+  hide() {
+    this.hidden = !this.hidden;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(HerbivoreAnimalThatWasInjuredByCarnivoresBite) {
```

Comment:
i think is too much for naming of variables  ))))

Reply:
fix @GarryRocksman comment
also, we use variable names that's started from the capital letter for the constants
you have just simple variable for storing some parameter, so you don't need to start it with the capital letter